README.md

```diff
@@ -40,6 +40,11 @@ You can backup a database from outside the container:
 $ docker exec postgresql pg_dump -Fc -U postgres <DBNAME> > /var/backups/<DBNAME>.pgdump
 ```
 
+You can restore a database from outside the container:
+```
+$ cat /var/backups/<DBNAME>.pgdump | docker exec -i postgresql pg_restore -Fc -U postgres -d <DBNAME>
```

Comment:
In my case, the pipe doesn't work without `-i`. Just hangs forever and nothing happens. Maybe you can try by yourself?